README.md

```diff
@@ -37,65 +37,134 @@ New features should be based on the `master` branch.
 
 Go Modules is used to manage dependencies.
 
-### Building
+### Go
+
+#### Building drone-gke executable
 
 ```bash
-go build
+make drone-gke
 ```
 
-### Testing
+#### Testing the executable
 
 ```bash
 go test
 ```
 
-### Container publishing
+### Docker
+
+#### Building images
 
-Build the Docker image with the following commands:
+Using a custom repo name:
 
+```sh
+docker_repo_name=your-docker-hub-user make docker-build
 ```
-OWNER=your-docker-hub-user ./bin/dist
+
+Using a custom tag:
+
+```sh
+docker_tag=alpha make docker-build
 ```
 
-## Usage
+#### Publishing images
 
-Using it in a `.drone.yml` pipeline: please take a look at [the docs](DOCS.md).
+Existing images:
 
-Executing locally from the working directory:
+```sh
+# runs docker push with custom repo, tag values
+docker_repo_name=your-docker-hub-user docker_tag=alpha make docker-publish
+```
+
+## All-in-one
 
+```sh
+# runs go build, docker build, docker push using defaults
+make dist
 ```
+
+## Usage
+
+### Running within a Drone pipeline
+
+Please take a look at [the docs](DOCS.md).
+
+### Running locally
+
+Using sample configurations within `local-example`:
+
+```sh
 # Deploy the manifest templates in local-example/
-$ cd local-example/
-
-# Set to the path of your GCP service account JSON file
-$ export GOOGLE_APPLICATION_CREDENTIALS=xxx
-
-# Set to your cluster
-$ export CLUSTER=yyy
-
-# Set to your cluster's zone
-$ export ZONE=zzz
-
-# The variables required for the templates in JSON format
-$ cat vars.json
-{
-   "app": "echo",
-   "env": "dev",
-   "image": "gcr.io/google_containers/echoserver:1.4"
-}
-
-# Execute the plugin
-$ docker run --rm \
```

Comment:
The reason why the docker run command verbose here is to show the command and match the format of most other drone plugins (https://github.com/drone-plugins/drone-git#usage, https://github.com/jmccann/drone-terraform#usage).

Hiding it in a make command is a departure from the ecosystem, and one would have to open the makefile to look at the example.

Reply:
ah wasn't aware of that! i'll revert back to that.

Reply:
Thanks!